packages/devtools_app/lib/src/logging/flutter/logging_screen.dart

```diff
@@ -53,16 +61,73 @@ class _LoggingScreenState extends State<LoggingScreenBody> {
         mainAxisAlignment: MainAxisAlignment.spaceBetween,
         children: [
           RaisedButton(
-            onPressed: _clearLogs,
             child: const Text('Clear logs'),
+            onPressed: _clearLogs,
           ),
           StructuredErrorsToggle(),
         ],
       ),
+      Expanded(
+        child: LogsTable(
+          data: controller.data,
+        ),
+      ),
     ]);
   }
 
   void _clearLogs() {
-    // TODO(https://github.com/flutter/devtools/issues/1286): do this.
+    setState(() {
+      controller.clear();
+    });
   }
 }
+
+class LogsTable extends StatefulWidget {
+  const LogsTable({Key key, this.data}) : super(key: key);
+  final List<LogData> data;
+
+  @override
+  _LogsTableState createState() => _LogsTableState();
+}
+
+class _LogsTableState extends State<LogsTable> {
+  final List<ColumnData<LogData>> columns = [
```

Comment:
uber nit: maybe define the colums in the LogsTable class instead of the State class as the columns are not mutable state (yet?)

Reply:
Done as part of above comment.